Resources/Prototypes/_Harmony/Accents/word_replacements.yml

```
@@ -0,0 +1,205 @@
+    accent-irish-words-156: accent-irish-words-replace-156
+    accent-irish-words-157: accent-irish-words-replace-157
+    accent-irish-words-158: accent-irish-words-replace-158
+    accent-irish-words-159: accent-irish-words-replace-159
+    accent-irish-words-160: accent-irish-words-replace-160
+    accent-irish-words-161: accent-irish-words-replace-161
+    accent-irish-words-162: accent-irish-words-replace-162
+    accent-irish-words-163: accent-irish-words-replace-163
+    accent-irish-words-164: accent-irish-words-replace-164
+    accent-irish-words-165: accent-irish-words-replace-165
+    accent-irish-words-165-2: accent-irish-words-replace-165
+    accent-irish-words-166: accent-irish-words-replace-166
+    accent-irish-words-167: accent-irish-words-replace-167
+    accent-irish-words-168: accent-irish-words-replace-168
+    accent-irish-words-169: accent-irish-words-replace-169
+    accent-irish-words-170: accent-irish-words-replace-170
+- type: accent
+  id: scandinavian
+  wordReplacements:
+    accent-scandinavian-words-1: accent-scandinavian-words-replace-1
+    accent-scandinavian-words-1-2: accent-scandinavian-words-replace-1
+    accent-scandinavian-words-2: accent-scandinavian-words-replace-2
+    accent-scandinavian-words-3: accent-scandinavian-words-replace-3
+    accent-scandinavian-words-4: accent-scandinavian-words-replace-4
+    accent-scandinavian-words-5: accent-scandinavian-words-replace-5
+    accent-scandinavian-words-6: accent-scandinavian-words-replace-6
+    accent-scandinavian-words-6-1: accent-scandinavian-words-replace-6
+    accent-scandinavian-words-7: accent-scandinavian-words-replace-7
+    accent-scandinavian-words-8: accent-scandinavian-words-replace-8
+    accent-scandinavian-words-9: accent-scandinavian-words-replace-9
+    accent-scandinavian-words-9-2: accent-scandinavian-words-replace-9
+    accent-scandinavian-words-10: accent-scandinavian-words-replace-10
+    accent-scandinavian-words-11: accent-scandinavian-words-replace-11
+    accent-scandinavian-words-12: accent-scandinavian-words-replace-12
+    accent-scandinavian-words-13: accent-scandinavian-words-replace-13
+    accent-scandinavian-words-14: accent-scandinavian-words-replace-14
+    accent-scandinavian-words-15: accent-scandinavian-words-replace-15
+    accent-scandinavian-words-16: accent-scandinavian-words-replace-16
+    accent-scandinavian-words-17: accent-scandinavian-words-replace-17
+    accent-scandinavian-words-18: accent-scandinavian-words-replace-18
+    accent-scandinavian-words-19: accent-scandinavian-words-replace-19
+    accent-scandinavian-words-20: accent-scandinavian-words-replace-20
+    accent-scandinavian-words-21: accent-scandinavian-words-replace-21
+    accent-scandinavian-words-22: accent-scandinavian-words-replace-22
+    accent-scandinavian-words-23: accent-scandinavian-words-replace-23
+    accent-scandinavian-words-24: accent-scandinavian-words-replace-24
```

Files should always end with one empty line
```suggestion
    accent-scandinavian-words-24: accent-scandinavian-words-replace-24

```